src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {
+      person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )
+    }
+    <p className="Person__partner">
+      { person.isMarried === true
+        ? (
+          <>
+            {`${person.partnerName} is my`}
+            {person.sex === 'm' ? ' wife' : ' husband'}
+          </>
+        )
+        : 'I am not married'
+      }
```

Comment:
move status to a variable `personStatus`
```suggestion
      {person.isMarried ?
        `${person.partnerName} is my ${personStatus}`
        : 'I am not married'
      }
```